game/scripts/vscripts/globals.lua

```diff
@@ -0,0 +1,13 @@
+-- I don't like this..
```

Comment:
I don't like this either. Make it a local inside of `gold.lua` since that's the only place it's ever used.

If another part of code needs to access this data it should be an API instead of accessing the data directly.